src/simsopt/objectives/fluxobjective.py

```diff
@@ -40,7 +40,10 @@ def __init__(self, surface, field, target=None):
     def J(self):
         n = self.surface.normal()
         Bcoil = self.field.B().reshape(n.shape)
-        Btarget = self.target if self.target is not None else []
+        if self.target is not None:
+            Btarget = np.ascontiguousarray(self.target)
+        else:
+            Btarget = []
```

Comment:
Is there any problem with moving these lines up to the `__init__()` function, replacing `Btarget` -> `self.Btarget`? Florian found that evaluating `J()` was performance-relevant enough to put the next line into C++, so it's possible that having this little bit of code re-run every time `J()` is evaluated might affect performance.